website/docs/docs/cloud/connect-data-platform/connnect-bigquery.md

```diff
@@ -52,6 +52,114 @@ As an end user, if your organization has set up BigQuery OAuth, you can link a p
 
 To learn how to optimize performance with data platform-specific configurations in dbt Cloud, refer to [BigQuery-specific configuration](/reference/resource-configs/bigquery-configs).
 
+### Optional configurations
+
+In BigQuery, optional configurations let you tailor settings for tasks such as running queries, loading or exporting data, and creating tables. While not necessary for basic operations, these options give you greater control over how BigQuery functions behind the scenes to meet your requirements.
+
+To customize your optional configurations in dbt Cloud:
+
+- Select your BigQuery project from the left-hand menu
+- Click **Settings** towards the top right
+- Scroll down to **Development Connection** and select BigQuery
+- Click **Edit** and then scroll down to **Optional settings**
+
+<Lightbox src="/img/bigquery/bigquery-optional-config.png" width="70%" title="BigQuery optional configuration"/>
+
+The following are the optional configs you can set in dbt Cloud:
```

Comment:
i suggest considering organizing these into a table with a value, info/description, type (string or integer), and example so it's easier for the user to understand what to add.

Reply:
hey @nataliefiann , unresolving this comment as it looks like there isn't a table that lists _all_ of the configs with clear info, examples, types (is it a string, is it an integer, etc.). so for example, this is a sample table that isn't fleshed out and would need tweaking if you plan on using it

```
|  config | info | type | example | 
| ------- | ----| ----- | -------- |
| max bytes billed | info here | integer | 100000 |
| priority | info here | string | interactive or batch |
…. rest of the configs…
```